telematic_system/telematic_cloud_messaging/src/main/java/com/telematic/telematic_cloud_messaging/nats_influx_connection/NatsInfluxPush.java

```diff
@@ -95,9 +95,16 @@ public void run() {
     }
 
     private void adjustConfig() {
-        config.influxUri = "http://" + config.influxUri + ":" + config.influxPort;
-        config.influxBucketType = Config.BucketType.valueOf(config.influxBucketTypeStr);
-        logger.info("Adjusted config: {}", config);
+        try{
+            config.influxUri = "http://" + config.influxUri + ":" + config.influxPort;
+            config.influxBucketType = Config.BucketType.valueOf(config.influxBucketTypeStr);
+            logger.info("Adjusted config: {}", config);
+        }catch(IllegalArgumentException ex){
+            logger.error("Invalid bucket type: {}. Error: {}", config.influxBucketTypeStr, ex.getMessage());
+        } catch (NullPointerException ex) {
+            logger.error("Bucket type cannot be null: {}. Error: {}", config.influxBucketTypeStr, ex.getMessage());
```

Comment:
Can we return valid options of bucket type with this error?

Reply:
Addressed https://github.com/usdot-fhwa-stol/cda-telematics/pull/205/commits/209ca50bbc8a936fa83b42095a178254dd813345